### ISSUE TITLE
Web example

### ISSUE THREAD

**Issue by XV02** [AnyChart/AnyChart#79]
Just the code in the example of the web page is wrong, doesn´t add data to the pie chart (Gradle example)

**Comment by Shestac92** [comment]
@XV02 
Please, can you provide the link to the wrong example? Do you mean Android library code snippets?

**Comment by XV02** [comment]
No, just the first simple example, it´s a little one, but I think that is
the main that the people use to test, so maybe you'll need to fix it.

The 'data' isn't asigned to 'Pie' and if someone use it, doesn't work.

https://www.anychart.com/es/technical-integrations/samples/android-charts/
[image: Captura.PNG]

**Comment by Shestac92** [comment]
@XV02 
Thank you for your report!
There's a line missing from the code snippet. The snippet code should be the following:
```
Pie pie = AnyChart.pie();

List<DataEntry> data = new ArrayList<>();
data.add(new ValueDataEntry("John", 10000));
data.add(new ValueDataEntry("Jake", 12000));
data.add(new ValueDataEntry("Peter", 18000));

pie.setData(data);

AnyChartView anyChartView = (AnyChartView) findViewById(R.id.any_chart_view);
anyChartView.setChart(pie);
```

We will correct the article as soon as possible!
